src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -429,6 +430,19 @@ define(function (require, exports, module) {
         callback();
     }
 
+    // Based on http://stackoverflow.com/questions/21797299/convert-base64-string-to-arraybuffer
```

Comment:
@timmoy there won't be any overhead because requires are cached. I agree with @humphd that this probably should be in FilerUtils.js instead. We can fix other issues with functions that aren't in there right now that should be in FilerUtils later.